server/backend/src/main/java/com/inventory/server/product/rest/Controller.java

```diff
@@ -3,6 +3,7 @@
 import com.inventory.server.product.ProductService;
 import com.inventory.server.product.model.Product;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
```

Comment:
As we have two controllers in this packages better rename this one to `ProductController`